src/PKSim.BatchTool/BatchRegister.cs

```diff
@@ -24,9 +23,6 @@ public override void RegisterInContainer(IContainer container)
          container.Register<IDiagramModelToXmlMapper, CLIDiagramModelToXmlMapper>();
          container.Register<IDiagramModel, CLIDiagramModel>();
          container.Register<IJournalDiagramManagerFactory, CLIJournalDiagramManagerFactory>();
-
-
-         container.Register(typeof(IInputAndOutputBatchView<>), typeof(InputAndOutputBatchView<>));
```

Comment:
I have removed this abstraction. It made life too complicated and we only have one implementation